dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -91,6 +91,9 @@
 
 = link_to t('crud.back'), :back
 
+- if current_user.can_edit_password? && current_user.encrypted_password.blank?
+  %div#add-password-fields
```

Comment:
It's a little odd that we've got the `migrated?` check below but not here.  This goes away if the JS handles the mountpoint, but otherwise maybe these should be parallel.

Reply:
good call. i added that to the form below later and forgot to add it here